ts/index.ts

```diff
@@ -1,28 +1,27 @@
-/* eslint-disable @typescript-eslint/ban-ts-comment */
-// @ts-ignore
-import { utils } from 'libp2p-interfaces/src/pubsub'
+import Pubsub, { InMessage, utils } from 'libp2p-interfaces/src/pubsub'
 import { MessageCache } from './message-cache'
 import {
   RPCCodec,
-  RPC, Message, InMessage,
+  RPC, Message,
```

Comment:
Might be good to move `Message` to libp2p-interfaces too.
Future PR

Reply:
Argh, we already have that in libp2p-interfaces just need to change import, but we can do that in separate PR if this PR is blocker for gossipsub migration?